### ISSUE TITLE
raw_response.description will lead to restclient/raw_response.rb:34:in `size': closed stream (IOError)

### ISSUE THREAD

**Issue by xxjapp** [rest-client/rest-client#676]
Test source is following:

```ruby
#!/usr/bin/env ruby
# encoding: utf-8

require 'rest-client'

raw_response = RestClient::Request.execute({
    method:         :get,
    url:            'https://www.google.com/',
    raw_response:   true,
    verify_ssl:     false
})

puts raw_response.description
```
The output was:

```bash
# ./test.rb
/usr/local/lib/ruby/gems/2.3.0/gems/rest-client-2.0.2/lib/restclient/raw_response.rb:34:in `size': closed stream (IOError)
        from /usr/local/lib/ruby/gems/2.3.0/gems/rest-client-2.0.2/lib/restclient/raw_response.rb:34:in `size'
        from /usr/local/lib/ruby/gems/2.3.0/gems/rest-client-2.0.2/lib/restclient/abstract_response.rb:113:in `description'
        from ./test.rb:13:in `<main>'
```

**Comment by xxjapp** [comment]
Now I use the following patch:

```ruby
module RestClient
    class RawResponse
        def size
            File.size file
        rescue IOError
            File.size file.path
        end
    end
end
```

**Comment by ab** [comment]
I believe this was fixed by de5e2685fda0daebb0049a14cd296c9785186d58, which is included in rest-client 2.1.0, released today.

Please feel free to reopen if it is still an issue.
